bedrock/mozorg/templates/mozorg/contact/includes/tab-nav.html

```diff
@@ -4,14 +4,16 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-<ul class="category-tabs" role="navigation">
-  <li {% if current == 'contact' %}class="current"{% endif %} data-id="contact">
-    <a href="{{ url('mozorg.contact.contact-landing') }}">Contact us</a>
-  </li>
-  <li {% if current == 'spaces' %}class="current"{% endif %} data-id="spaces">
-    <a href="{{ url('mozorg.contact.spaces.spaces-landing') }}">Spaces</a>
-  </li>
-  <li data-id="communities">
-    <a href="https://community.mozilla.org/groups/">Communities</a>
-  </li>
-</ul>
+<nav>
```

Comment:
**nit**: when there are multiple navigations on the page, it can help to distinguish them with `aria-label` https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role#labeling_landmarks

```suggestion
<nav aria-label="Category">
```